set.mm

```diff
@@ -814993,14 +815222,293 @@ two graphs which are not (ordinary) isomorphisms between these graphs, as
   ${
     $d G i $.  $d H i $.
     $( Isomorphic hypergraphs are locally isomorphic.  (Contributed by AV,
-       12-Jun-2025.) $)
+       12-Jun-2025.)  (Proof shortened by AV, 11-Jul-2025.) $)
     gricgrlic $p |- ( ( G e. UHGraph /\ H e. UHGraph )
                       -> ( G ~=gr H -> G ~=lgr H ) ) $=
-      ( vi cgric wbr cuhgr wa cgrlic cgrim co c0 wne wi brgric biimpi cv wex n0
-      wcel w3a cgrlim simp3l simp3r simp1 uhgrimgrlim syl3anc brgrilci syl 3exp
-      exlimiv sylbi mpcom com12 ) ABDEZAFSZBFSZGZABHEZABIJZKLZUNUQURMZUNUTABNOU
-      TCPZUSSZCQUNVAMZCUSRVCVDCVCUNUQURVCUNUQTZVBABUAJSZURVEUOUPVCVFVCUNUOUPUBV
-      CUNUOUPUCVCUNUQUDVBABUEUFABVBUGUHUIUJUKULUM $.
+      ( vi cgric wbr cuhgr wcel wa cgrlic cgrim co c0 wne wi brgric cv n0 sylbi
+      wex w3a cgrlim uhgrimgrlim brgrilci syl 3expa expcom exlimiv com12 ) ABDE
+      ZAFGZBFGZHZABIEZUIABJKZLMZULUMNZABOUOCPZUNGZCSUPCUNQURUPCULURUMUJUKURUMUJ
+      UKURTUQABUAKGUMUQABUBABUQUCUDUEUFUGRRUH $.
+  $}
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Triangles in graphs
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+   In this subsection, a triangle is defined as a set of three vertices of a
+   given graph.  Therefore, a trinagle is not a graph (a "triangle graph", see
+   Wikipedia "Triangle graph", ~ https://en.wikipedia.org/wiki/Triangle_graph ,
+   27-Jul-2025: "In the mathematical field of graph theory, the triangle graph
+   is a planar undirected graph with 3 vertices and 3 edges, in the form of a
+   triangle."  Furthermore, "The triangle graph is also known as the cycle
+   graph C_3 and the complete graph K_3."  However, a triangle
```

Comment:
The way this is organized is a bit confusing, as you first give a definition of triangle for this section, and immediately afterwards give a Wikipedia citation where "triangle" is used with a different meaning.

Maybe it would be clearer if you first gave the general definition of triangle (graphs), then stated that in this chapter, triangles are defined as a (completely connected) set of vertices.

Reply:
With my latest commit, I provide a completely revised section header comment: typo fixed, the order of the definitions changed, and 2 bibliographic references added.
Furthermore, I mentioned your proposed alternate, more general definition, so that someone else could use it if needed.